### PR TITLE
Remove needrestart DPKG hook

### PR DESCRIPTION
**Pull request by SlothOfAnarchy**
closes #15

### CODE REVIEW

File: tasks/upgrade_debian.yml [hunk 1]
```diff
@@ -21,6 +21,11 @@
   debug: msg="{{ updates.stdout_lines | count }} packages to upgrade ({{ updates.stdout_lines | join(', ') }})"
   when: updates.stdout_lines | count > 1
 
+- name: Remove needrestart DPKG hook
```
Comment (Mr-Pi) [inline tasks/upgrade_debian.yml]: Please move this section to line 13, direct after the 'Install dependencies' task.

**Review by Mr-Pi — CHANGES_REQUESTED** [review]

File: tasks/upgrade_debian.yml (hunk 1 above)
Comment (SlothOfAnarchy) [reply tasks/upgrade_debian.yml]: Why? I placed it specifically directly before executing the `apt` task.

Comment (Mr-Pi) [reply tasks/upgrade_debian.yml]: I think is more appropriate to configure the system first(such thinks like installing all dependencies, write correct config files...) and do the regular stuff afterwards. Even more I thinks it's more clear to have all upgrade tast in a sequence, without any configuration in there.

Comment (SlothOfAnarchy) [reply tasks/upgrade_debian.yml]: Fine.

**Review by Mr-Pi — APPROVED** [review]

**Pull request merged**